migtests/scripts/compare-html-reports.py

```diff
@@ -129,44 +133,85 @@ def generate_diff_list(list1, list2, section_name, file1_path, file2_path):
 def dict_to_list(dict_data):
     """Convert dictionary to list of formatted strings."""
     return [f"{k} -> {v}" for k, v in dict_data.items()]
+    
+def compare_html_tags(html_data1, html_data2):
```

Comment:
Comparing unique tags is one.

How about one enhancement - also match the counts for each tags(number of times each tag appeared)
I guess it should be small change, lib might be providing that.